prepare/cli.py

```diff
@@ -0,0 +1,160 @@
+import click
+
+from numpy import unique, ndarray
+from scipy.sparse.csc import csc_matrix
+from os.path import isfile, isdir, splitext, expanduser
+
+settings = dict(help_option_names=['-h', '--help'])
+
+
+@click.command()
+@click.argument('dataset', nargs=1, metavar='<dataset: file or path to data>', required=True)
+@click.option('--layout', default='umap', type=click.Choice(['umap', 'tsne', 'umap+tsne']),
```

Comment:
could layout take more than one option instead of the umap+tsne? Then we could loop though? This solves a possible permutation issue if we have more than two layout options.

Reply:
great catch, changed the argument type to handle this